lib/Settings/Display.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:cboard_mobile/shared/app-bar.dart';
+import 'package:flutter/material.dart';
+
+class Display extends StatelessWidget {
+// static const String route = "display";
+// final Map<String,String> arguments;
+// Display(this.arguments);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: BaseAppBar(
+          title: Text('Display'),
+          appBar: AppBar(),
+        ),
+        body: Center(
+          child: MyStatefulWidget(),
+        ));
+  }
+}
+
+enum size { Standard, Large, ExtraLarge }
+
+class MyStatefulWidget extends StatefulWidget {
+  MyStatefulWidget({Key key}) : super(key: key);
+  @override
+  _MyStatefulWidget createState() => _MyStatefulWidget();
+}
+
+class _MyStatefulWidget extends State<MyStatefulWidget> {
+  bool _darkTheme = false;
+  bool _predictiveText = false;
+  bool _outputBar = false;
+  size _uiSize = size.Standard;
+  size _fontSize = size.Standard;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView(
+      children: <Widget>[
+        radio_section('UI Size', 'Elements Size', _uiSize),
+        divider(),
+        radio_section('Font Size', 'App Font Size', _fontSize),
+        divider(),
+        listTile_section(
+            'Enable Dark Theme',
+            'The theme specifies the color of the components, darkness of the surfaces, level of shadow, appropriate opacity of ink elements, etc.',
+            _darkTheme),
+        divider(),
+        listTile_section(
+            'Enable Predictive Text',
+            'Predictive text will arrange the cards so that recommended cards based on past history will be recommended and shown first',
+            _predictiveText),
+        divider(),
+        listTile_section(
+            'Hide the Output Bar',
+            'Hides the white bar on the top where you build a sentence',
+            _outputBar),
+      ],
+    );
+  }
+
+  Widget divider() {
+    return Divider(
+      height: 20,
+      color: Color(0xffE0E0E0),
+      thickness: 1,
+    );
+  }
+
+  Widget listTile_section(String title, String subtitle, bool state) {
+    return SwitchListTile(
+      activeColor: Color(0xff5600E8),
+      title: Container(
+          width: 196,
+          child:
+              Text(title, style: TextStyle(fontSize: 14, color: Colors.black))),
+      subtitle: Text(subtitle,
+          style: TextStyle(fontSize: 12, color: Color(0xff7C7C7C))),
+      value: state,
+      onChanged: (bool newValue) {
+        setState(() {
+          if (title == "Enable Dark Theme") {
+            _darkTheme = newValue;
+          } else if (title == "Enable Predictive Text")
+            _predictiveText = newValue;
+          else
+            _outputBar = newValue;
+        });
```

Comment:
Hmm... it looks like all the `SwitchTileSections` just toggle between two values - instead of passing in a value and having this if-else condition, would be possible to directly pass in an `onChanged` function that modifies some values instead? This might help simplify some logic here.